packages/gatsby-graphiql-explorer/src/app/index.ejs

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>GraphiQL</title>
+    <meta name="robots" content="noindex" />
+    <meta name="referrer" content="origin" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+  </head>
+  <body>
+    <div id="root" class="graphiql-container">Loading...</div>
+    <script
+      type="text/javascript"
+      src="/___graphql/<%= htmlWebpackPlugin.files.chunks.main.entry %>?<%= htmlWebpackPlugin.files.chunks.main.hash %>"
```

Comment:
Would this cause issues with #14234 (e.g. if navigating to /_graphql)? To avoid, we could just redirect to ___graphql, instead of exposing other endpoints--which is _probably_ a better solution anyways.

Reply:
as long as `/___graphql` is handled by gatsby develop - this is fine regardless of which of the variants we use. Initially wanted to make this relative, but it would require trailing slash after `/___graphql/` to work

Reply:
I can also adjust e2e test, to actually do something in IDE and verify that it's working (and not only check title as it works right now)

Reply:
e2e tests added